### PR TITLE
Dockerise

### PR DESCRIPTION
**Pull request by TheCodeYoda**
Dockerise the ml server

### CODE REVIEW

File: ml/generate_image_index.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+import os
+
+BASEURL = "http://0.0.0.0:5001/process_image"
+IMG_PATH = "../data/images/"
+imgs = [x for x in os.listdir(IMG_PATH) if x.split('.')[-1] in ['jpg', 'png', 'jpeg']]
+img_index = []
+for img in tqdm(imgs):
```
Comment (nuwandavek) [inline ml/generate_image_index.py:9]: Have to import tqdm

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+services:
+  scanpix-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: scanpix-server:1.0
+    ports:
+      - "5001:5001"
+    volumes:
+      - "./data:/scanpix/data"
```
Comment (nuwandavek) [inline docker-compose.yml:10]: Can you bring up both dockers as services in this compose file?

**Comment by nuwandavek** [comment]
Looks great! We can merge this after resolving the minor issues!

File: docker-compose.yml (hunk 2 above)
Comment (TheCodeYoda) [reply docker-compose.yml:10]: Should we dockerise the frontend though? Lets keep the electron app running locally.

Comment (nuwandavek) [reply docker-compose.yml:10]: Yeah, makes sense. Just add a bash script that brings up the backend and runs the front end. Good to have a single command, I think.

**Review by sam17 — APPROVED** [review]

**Pull request merged**